src/app/amsterdam/projecten/stadsdeel/page.tsx

```diff
@@ -81,8 +83,8 @@ function Project() {
               ))}
             </LinkList>
           </Grid.Cell>
-          <Grid.Cell span={{ narrow: 1, medium: 2, wide: 4 }} start={{ narrow: 4, medium: 7, wide: 8 }}>
-            <Heading color="inverse" level={2} size="level-4" className="ams-mb-s">
+          <Grid.Cell span={{ narrow: 4, medium: 2, wide: 4 }} start={{ narrow: 1, medium: 7, wide: 8 }}>
+            <Heading color="inverse" level={2} size="level-4" className="ams-mb-m">
```

Comment:
Small after Heading 4, also matches the one for ‘Werk aan de weg’ to its left:

```suggestion
            <Heading color="inverse" level={2} size="level-4" className="ams-mb-s">
```